gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 #Mon Sep 19 08:21:35 CEST 2022
 distributionBase=GRADLE_USER_HOME
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-bin.zip
```

Comment:
Do not update gradle into a single PR, this can cause issues on the other working branches if the order of the merge of the pull requests doesn't respect the order of PR opening. And again this type of operation needs to be made in a branch used only for this purpose.